docs/contributing/how_to_template.ipynb

```diff
@@ -0,0 +1,200 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5e8cf35c",
+   "metadata": {},
+   "source": [
+    "# How-to template"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3da983f5",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Note\n",
```

Comment:
I don't think there is a need for a note. Make "Step One" in the how-to template something like

## Setting up the how-to guide
1. filename and location ...
2. Replace the title ...
3. Fill out the "Before you start" section ...